### ISSUE TITLE
Check if games sandbox exists before starting server

### ISSUE THREAD

**Issue by seanyesmunt** [lbryio/lbry-desktop#2251]
## The Issue
With the new sandbox server, it's not possible to run two apps at once. You get an error saying the sandbox port is already in use.

### Steps to Reproduce

1. Run the app in dev mode 
2. Run the latest rc
3. See the error

### Suggested Solutions

Check if the port is in use before starting the server.

**Comment by skhameneh** [comment]
🤔 seems reasonable, I'd imagine this is rarely needed (except maybe in your case of testing/dev multiple instances)

**Comment by seanyesmunt** [comment]
Yep. It's nice to have the app and the dev build open at the same time for testing.

**Comment by seanyesmunt** [comment]
Or maybe only start the sandbox server with a flag passed in in dev mode? Then it would default to off and the production app would still work

**Comment by skhameneh** [comment]
@seanyesmunt fixed this
